### PR TITLE
Adapt documentation to CTK interface change

### PR DESCRIPTION
**Pull request by marcus-wirtz-snkeos**
Interface of CTK class `ctk.ctkDICOMQuery()` for retreiving queried InstanceUIDs has changed from `ctk.ctkDICOMQuery().studyInstanceUIDQueried` to `ctk.ctkDICOMQuery().studyAndSeriesInstanceUIDQueried`.

Cf. https://github.com/commontk/CTK/commit/34a390a49bc8e41e2ff8a6fe72382974c0787ad7

### CODE REVIEW

**Review by lassoan — DISMISSED** [review]
Thank you, good catch. I've updated the retrieve part as well (retrieve the series UID from the study/series pair and use that to retrieve the series).

File: Docs/developer_guide/script_repository/dicom.md [hunk 1]
```diff
@@ -260,15 +263,15 @@ dicomRetrieve.callingAETitle = dicomQuery.callingAETitle
...
+dicomRetrieve.setMoveDestinationAETitle("SLICER")
 dicomRetrieve.setDatabase(slicer.dicomDatabase)
-for study in dicomQuery.studyInstanceUIDQueried:
-  print(f"ctkDICOMRetrieveTest2: Retrieving {study}")
+for study, series in dicomQuery.studyAndSeriesInstanceUIDQueried:
+  print(f"ctkDICOMRetrieveTest: Retrieving {study}")
   slicer.app.processEvents()
   if dicomQuery.preferCGET:
-    success = dicomRetrieve.getStudy(study)
+    success = dicomRetrieve.getSeries(series)
```
Comment (marcus-wirtz-snkeos) [inline Docs/developer_guide/script_repository/dicom.md:272]: @lassoan I think the getSeries(), moveSeries() needs both, the `study` key and the `series` key

Comment (lassoan) [reply Docs/developer_guide/script_repository/dicom.md]: That's correct. Somehow I missed this when I copied the changes from my Python console. I've updated it now. Please test.

**Review by lassoan — APPROVED** [review]
Looks all good then.

**Comment by lassoan** [comment]
Thanks for your contribution @marcus-wirtz-snkeos !

**Pull request merged**